src/senior/SeniorModel.coffee

```diff
@@ -1,27 +1,39 @@
 class @SeniorModel
-  _events: {}
-  isShowModal: false
-  data: [
-    { name: 'AAA', amount: 2000 }
-    { name: 'BBB', amount: 10 }
-    { name: 'CCC', amount: 10000 }
-  ]
-
-  get: (key) ->
-    @[key]
-
-  set: (key, value, silent) ->
-    @[key] = value
-    if not @_events[key] || silent
-      return
-    for callback in @_events[key]
-      (callback)()
-
-  onChange: (key, callback) ->
-    @_events[key] = @_events[key] || []
-    @_events[key].push(callback)
+  constructor: ->
+    @_events = {}
+    @_isShowModal = false
```

Comment:
Probably, `isShowModal` is invalid as English.
For example, `isModalShowing` is better than it, by the [GitHub code search](https://github.com/search)!